highcharts/src/main/java/us/ascendtech/highcharts/client/chartoptions/rangeselector/Buttons.java

```diff
@@ -0,0 +1,88 @@
+package us.ascendtech.highcharts.client.chartoptions.rangeselector;
+
+import elemental2.core.JsArray;
+import jsinterop.annotations.JsOverlay;
+import jsinterop.annotations.JsPackage;
+import jsinterop.annotations.JsProperty;
+import jsinterop.annotations.JsType;
+
+
+/**
+ * @author Samuel Palacean
+ * Docs: https://api.highcharts.com/highstock/rangeSelector.buttons
+ */
+@JsType(isNative = true, name = "Object", namespace = JsPackage.GLOBAL)
+public class Buttons {
+    private double count;
+    private double offsetMax;
+    private double offsetMin;
+    private Boolean preserveDataGrouping;
```

Comment:
Why not primitive type `boolean`?
Are you also handling null values?

Reply:
same as above